presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergLogicalPlanner.java

```diff
@@ -112,6 +113,54 @@ protected QueryRunner createQueryRunner()
         return createIcebergQueryRunner(ImmutableMap.of("experimental.pushdown-subfields-enabled", "true"), ImmutableMap.of());
     }
 
+    @Test
+    public void testFiltersWithPushdownDisable()
+    {
+        Session sessionWithoutFilterPushdown = pushdownFilterDisabled();
```

Comment:
Since the filter pushdown session property is disabled by default, I think this is not required.

Reply:
Fixed, please take a look!